recipes/entt/3.x.x/conanfile.py

```diff
@@ -73,7 +73,3 @@ def package_info(self):
         self.cpp_info.set_property("cmake_target_name", "EnTT::EnTT")
         self.cpp_info.bindirs = []
         self.cpp_info.libdirs = []
-
-        # TODO: to remove in conan v2
-        self.cpp_info.names["cmake_find_package"] = "EnTT"
-        self.cpp_info.names["cmake_find_package_multi"] = "EnTT"
```

Comment:
Thanks a lot for the contribution @RazielXYZ.
One thing, this should be left in the recipe, it's not required to add those for new recipes but we are not removing this for the moment to not break users that may use the legacy generators in Conan 1.X. Some time in the future we will remove this from recipes.

Reply:
Ah, makes sense, I'll put it back